cowprotocol/explorer
# Add AppData row with Metadata Info in Orders details page

# Summary

Closes #157

Added a new row in order details page table in order to show metadata info

<img width="895" alt="image" src="https://user-images.githubusercontent.com/3328006/179066912-52da2911-afad-4e77-8ca2-aff9902f214d.png">

# To Test

1. Go to any order page. i.e `/orders/0xbc64cbd886df071a7076c189df0a04855a03858bf99d9c2ae01d6d48e618858984e5c8518c248de590d5302fd7c32d2ae6b0123c627272d7`
* You'll see metadata information next to `appData` row name in order details table

**NOTE:** 
In the image above, the `appData` shown is not related to the order since it was harcoded for testing purposes (I used this appData hash `0x5ddb2c8207c10b96fac92cb934ef9ba004bc007a073c9e5b13edc422f209ed80`. I've been trying some orders from `mainnet` network, but all of them returns timeout errors when use those `appData` values. 

FYI, you can check if the appData value has associated an `appDataDocument` in this working [example](https://codepen.io/pacara/pen/BadZgjg)
